src/language/libraries.md

```diff
@@ -92,34 +92,25 @@ import 'package:lib2/lib2.dart' hide foo;
 <a id="deferred-loading"></a>
 #### Lazily loading a library
 
-_Deferred loading_ (also called _lazy loading_)
-allows a web app to load a library on demand,
-if and when the library is needed.
-Here are some cases when you might use deferred loading:
+*Deferred loading* (also called *lazy loading*) allows
+web and Flutter Android apps to load a library on demand.
 
-* To reduce a web app's initial startup time.
+Use deferred loading in the following use cases:
+
+* To reduce an Android or web app's initial startup time.
 * To perform A/B testing—trying out
-  alternative implementations of an algorithm, for example.
+  alternative implementations of an algorithm.
 * To load rarely used functionality, such as optional screens and dialogs.
 
-{{site.alert.warn}}
-  **Only `dart compile js` supports deferred loading.**
-  Flutter and the Dart VM don't support deferred loading.
-  To learn more, see
-  [issue #33118](https://github.com/dart-lang/sdk/issues/33118) and
-  [issue #27776.](https://github.com/dart-lang/sdk/issues/27776)
-{{site.alert.end}}
-
-To lazily load a library, you must first
-import it using `deferred as`.
+To lazily load a library, import it using `deferred as`.
```

Comment:
AFAIK the flutter-AOT deferred loading solution will still bundle the deferred components in e.g. the Android APK. So they will not lazily be downloaded from network.

This is quite different from JS, where the actual JS files will be downloaded when needed.

It may be worth pointing out this aspect.